source/includes/_endpoint-dataset.md

```diff
@@ -486,6 +486,16 @@ in the API.
 
 Categorical variables will be exported with their name instead of their value.
 
```

Comment:
How onerous is moving this to POST with progress? The 'which variables to include' and a bunch of options is so much like what we want for tabbook options that they should be as similar as possible